docs/UserGuide.md

```diff
@@ -293,45 +364,75 @@ Example:
 CLI-nic data are saved in the hard disk automatically after any command that changes the data.
 There is no need to save manually.
 
-### Updating the quantity and/or tags of a product for a supplier or warehouse: `update`
+### Updating the quantity and/or tags of a product sold by a supplier: `update`
 
-Updates the quantity and/or tags of the product with the specified name in the specified supplier or warehouse.
-If the product does not exist for that supplier or warehouse, a new product will be created for that supplier
-or warehouse. 
+Updates the quantity and/or tags of the product with the specified name at the specified supplier index.
+If the product does not exist, a new product will be created for that supplier. 
 
-Format:	`update ct/TYPE n/NAME pd/PRODUCT_NAME [q/QUANTITY] [t/TAG]`
+Format:	`update ct/s i/INDEX pd/PRODUCT_NAME [q/QUANTITY] [t/TAG]`
 
-* `TYPE` specified should be either `s` for supplier or `w` for warehouse.
-* `NAME` and `PRODUCT_NAME` specified are case-insensitive.
-* The supplier or warehouse should currently exist in the CLI-nic application.
+* `INDEX` must be a positive integer, not exceeding the total length of the displayed supplier list in the GUI.
+* `PRODUCT_NAME` specified is case-insensitive.
+* The supplier should currently exist in the CLI-nic application.
 * `QUANTITY` should be a non-negative unsigned integer.
 * `TAG` should be a single alphanumeric word.
-* If the `PRODUCT_NAME` already exists in the supplier or warehouse, at least one optional argument has to be entered.
+* If `PRODUCT_NAME` already exists in the supplier, at least one optional argument has to be entered.
 
 Example:
 
-* `update ct/w n/WarehouseA pd/Panadol q/10 t/fever` : Updates the quantity of `Panadol` in `WarehouseA` to `10`
-  and gives `Panadol` a tag of `fever`.
+* `update ct/s i/4 pd/Panadol q/10 t/fever` : Updates the quantity of `Panadol` sold by the supplier at index 4 in the
+  list of displayed suppliers in the GUI to `10` and gives `Panadol` a tag of `fever`.
+  
+![update warehouse product](images/updateWarehouseProduct.png)
+
+### Updating the quantity and/or tags of a product stored in a warehouse: `update`
+
+Updates the quantity and/or tags of the product with the specified name at the specified warehouse index.
+If the product does not exist, a new product will be created for that warehouse. 
+
+Format:	`update ct/w i/INDEX pd/PRODUCT_NAME [q/QUANTITY] [t/TAG]`
```

Comment:
Are the blank spaces meant to be?

Reply:
I think is typo cos the rest don't have it

Reply:
Ahh I just checked! I think this is an issue with the rendering here! There are no extra blank spaces in the file!